cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1,611 @@
+# <a id='how-to-guides'>How-to-guides
+
+<!-- Intro to this point -->
+
+<!-- ## <a id='service-account-usage'> --service-account usage -->
+
+## <a id='custom-registry'> Custom registry credentials
+
+Apps CLI plug-in allows users to push images to their private registry by setting some flags when creating from local source code. 
+User can either trust a custom certificate in their system or pass the path to the certificate through flags.
```

Comment:
A user can either trust a custom certificate on their system or pass the path to the certificate via flags.